client/src/views/Simulation/components/VariablesPanel.vue

```diff
@@ -6,11 +6,16 @@
         :title="countersTitle"
         :data="countersData"
       />
-      <div slot="right">
-        <button type="button" disabled class="btn btn-secondary btn-settings">Settings</button>
-        <button type="button" class="btn btn-secondary py-0 btn-settings" @click="$emit('expand')">
-          <font-awesome-icon :icon="['fas', (expanded ? 'compress-alt' : 'expand-alt')]" />
-        </button>
+      <div class="d-flex align-items-center" slot="right">
+        <small class="mr-3 run-counter" v-if="getVariablesRunsCount">
+          {{`${getVariablesRunsCount} run${getVariablesRunsCount > 1 ? 's' : ''}`}}
+        </small>
+        <div class="btn-group">
```

Comment:
@RosaRomeroGomez asked to separate those buttons.
If it's fine to keep them grouped, we should fo that in all panels.

Reply:
whoops, i connected those buttons by mistake using the `btn-group` class